src/test/java/net/snowflake/client/jdbc/ResultSetStructuredTypesLatestIT.java

```diff
@@ -107,6 +108,17 @@ public void testMapStructAllTypes() throws SQLException {
     testMapAllTypes(true);
   }
 
+  @Test
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
```

Comment:
do we need this annotation? other structured types tests run already on github, correct?